build.gradle.kts

```diff
@@ -54,4 +55,11 @@ publishing {
             }
         }
     }
+}
+
+tasks.withType<Jar>() {
+    duplicatesStrategy = DuplicatesStrategy.EXCLUDE
+    configurations["compileClasspath"].forEach { file: File ->
+        from(zipTree(file.absoluteFile))
```

Comment:
What is the purpose of these changes here?
Looks like a way of making a "fat JAR" to me.

Reply:
It was added because when I have changed this implementation (in translator):

> implementation("com.github.vpa-research:libsl-parser:853ca7de04")


to this:

>     implementation(files("/Users/alekseevaroslav/Downloads/libsl-parser/build/libs/libsl-1.0-SNAPSHOT.jar"))


It doesn't find dependencies of the translator;

Reply:
"Fat" libraries may cause problems: if there are multiple versions of such libraries are being used simultaneously, then there is a high chance they will be loaded incorrectly or conflict with each-other.

Reply:
Fixed